lib/code_corps/map_utils.ex

```diff
@@ -8,7 +8,6 @@ defmodule CodeCorps.MapUtils do
   def keys_to_string(map), do: stringify_keys(map)
 
   # Goes through a list and stringifies keys of any map member
-  defp stringify_keys(nil), do: nil
   defp stringify_keys(%DateTime{} = val), do: val
```

Comment:
I think my point here was misunderstood.

Doing any of the following is the same as doing `fn(val), do: val`:

- `fn(%DateTime{} = val), do: val`
- `fn(nil = val), do: val`
- `fn(RandomApp.RandomModule = val), do: val`

All of these are equivalent _if_ all we're doing is passing along `val`.